ext/web/benches/timers_ops.rs

```diff
@@ -21,9 +21,9 @@ impl deno_web::TimersPermission for Permissions {
 
 fn setup() -> Vec<Extension> {
   vec![
-    deno_webidl::init_esm(),
-    deno_url::init_ops_and_esm(),
-    deno_console::init_esm(),
+    deno_webidl::deno_webidl::init_esm(),
+    deno_url::deno_url::init_esm(),
```

Comment:
I'm thinking we should leave function names as `init_ops_and_esm` and `init_ops`. Thoughts on that?

Reply:
That's totally fine as well. I can circle back and update all those names.